### DISCUSSION TITLE
If I push the big dataset into repository with .png format. Is it influencing the linguist to work?

### DISCUSSION THREAD

**Discussion by PinkR1ver** [github-linguist/linguist#5882] [Q&A] [answered]
Like the title shown.
https://github.com/PinkR1ver/Radiogenemics--on-Ivy-Gap/tree/master

**Comment by lildude** [comment, marked as answer]
png files are ignored as they're binary files. What will likely however stop analysis is the sheer number of files in the repo (wow! You have a lot!) as there's a hard limit. You're hitting the same thing as https://github.com/github/linguist/discussions/5878 from a few days ago

**Reply by PinkR1ver** [reply]
I see. Thanks, bro. :)
